cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -263,31 +264,16 @@ def upload_rna_coverage_bigwig_to_scout(self, case_id: str, dry_run: bool) -> No
                 )
 
             LOG.info(f"RNA coverage bigwig file {rna_coverage_bigwig.path} found")
-            dna_sample_id: str
-            dna_cases: List[str]
-            dna_sample_id, dna_cases = rna_dna_sample_case_map[rna_sample_id].popitem()
-            for dna_case_id in dna_cases:
-                if self.status_db.get_case_by_internal_id(internal_id=dna_case_id).is_uploaded:
-                    LOG.info(
-                        f"Uploading RNA coverage bigwig file for {dna_sample_id} in case {dna_case_id} in scout"
-                    )
-
-                    if dry_run:
-                        continue
-
-                    self.scout_api.upload_rna_coverage_bigwig(
-                        file_path=rna_coverage_bigwig.full_path,
-                        case_id=dna_case_id,
-                        customer_sample_id=dna_sample_id,
-                    )
-                    LOG.info(
-                        f"Uploaded RNA coverage bigwig file for {dna_sample_id} in case {dna_case_id}"
-                    )
-                else:
-                    LOG.warning(
-                        f"Upload of RNA coverage bigwig file for {dna_sample_id} in case {dna_case_id} skipped - "
-                        f"case has not finished uploading"
-                    )
+            successful_uploads_summary.append(
+                self._upload_splice_junctions_bed_or_coverage_for_rna_sample(
+                    file=rna_coverage_bigwig,
+                    rna_sample_id=rna_sample_id,
+                    rna_dna_sample_case_map=rna_dna_sample_case_map,
+                    dry_run=dry_run,
+                    is_splice_junctions=False,
+                )
+            )
```

Comment:
Similar logic used twice use a function to DRY.

```suggestion

-            def return_succesful_uploads_summary() -> List[str]:

                successful_uploads_summary: List[str] = []
                <<<other logic here >>> 
                return successful_uploads_summary
                
   
```

Reply:
Fixed